cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1328,6 +1329,11 @@ def setUp(self):
         super(TestComponentTemplates, self).setUp()
         self.templates = get_component_templates(self.course)
 
+        # Initialize the deprecated modules settings with empty list
+        XBlockDeprecatedAdvancedComponentConfig.objects.create(
```

Comment:
This is needed because if I add a settings entry for a test and then in another test if I fetch the model, settings that I set in previous test will be fetched. We need fresh settings every time.